### PR TITLE
Add konva@1.6.2 with git auto-update

### PR DESCRIPTION
**Pull request by clairetsai818**
Pull request for issue: #10964

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
* Git repository (required):https://github.com/konvajs/konva
* Official website (optional, not the repository):https://konvajs.github.io/
* NPM package url (optional):https://www.npmjs.com/package/konva
* License and its reference:[MIT](https://github.com/konvajs/konva/blob/master/package.json)
* GitHub / Bitbucket popularity (required):
- Count of watchers:84
- Count of stars:1241
- Count of forks:120
* NPM download stats (optional):
- Downloads in the last day:468
- Downloads in the last week:2140
- Downloads in the last month:9664

# Essential checklist
* [ ] I'm the author of this library
* [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] No already exist / duplicated issue and PR
* [x] The lib has notable popularity
* [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
* [x] More than 500 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

# Git commit checklist
* [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
* [x] The parent of the commit(s) in the PR is not old than 3 days.
* [x] Pull request is sending from a non-master branch with meaningful name.
* [x] Separate unrelated changes into different commits.
* [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [x] Close corresponding issue in commit message
* [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/konva/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "author": "Anton Lavrenov",
+  "description": "Konva is an HTML5 Canvas JavaScript framework that enables high performance animations, transitions, node nesting, layering, filtering, caching, event handling for desktop and mobile applications, and much more.",
+  "filename": "konva.min.js",
+  "keywords": [
+    "canvas",
+    "animations",
+    "graphic",
+    "html5"
+  ],
+  "homepage": "http://konvajs.github.io/",
```
Comment (sufuf3) [inline ajax/libs/konva/package.json]: `https://konvajs.github.io/`  please use https.

File: ajax/libs/konva/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "url": "git://github.com/konvajs/konva.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/konvajs/konva.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "konva*.js"
```
Comment (sufuf3) [inline ajax/libs/konva/package.json]: how about `konva*+(js|css|map)` ?

**Comment by clairetsai818** [comment]
I modified the filemap, please help me review this. Thanks!

**Comment by sufuf3** [comment]
@clairetsai818 homepage should use https `https://konvajs.github.io/`

**Comment by clairetsai818** [comment]
@sufuf3 I modified the homepage info, please help me review this, thanks!

File: ajax/libs/konva/package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    "url": "git://github.com/konvajs/konva.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/konvajs/konva.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "konva*.(js|css|map)"
```
Comment (sufuf3) [inline ajax/libs/konva/package.json]: `konva*.+(js|css|map)`
you forgot the `+`

**Comment by clairetsai818** [comment]
@sufuf3 I modified the filemap, please help me review this. Thanks!

**Comment by sufuf3** [comment]
Ping @extend1994 @hare1039

**Pull request merged**